demo_modules/chicago-brick-live/.jshintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "../.jshintrc",
+  // Enable Function to be used.  
```

Comment:
This is gone now, there is a new code to run sandboxed client code (similar to safeEval).  Right now this is in the module, but could be moved to /lib if it would be generally useful.